contracts/btc-staking/src/state/config.rs

```diff
@@ -49,4 +49,5 @@ pub struct Params {
     /// expressed as a decimal (e.g. 0.5 for 50%).
     #[derivative(Default(value = "String::from(\"0.1\")"))]
     pub slashing_rate: String,
+    pub rewards_denom: String,
```

Comment:
Can this be queried from Cosmos SDK or be retrieved [here](https://github.com/babylonlabs-io/babylon-contract/blob/50fb1d70bfbbf3a24e0e60b59ab8cf40429d623f/contracts/babylon/src/state/config.rs#L26-L27)?

Reply:
Fixed. Queried using the `query_bonded_denom`, which makes sense for consistency with other parts of the code.

That said, rewards denom and bonded denom are not necessarily the same. This is something we will need to address at some point.